bin/build.ts

```diff
@@ -43,11 +44,21 @@ main(process.argv)
   .then(exitCode => {
     process.exit(exitCode)
   })
-  .catch(e => {
-    if (e instanceof BuildErrorOutput) {
-      console.error(e.message)
+  .catch(error => {
+    if (error instanceof BuildErrorOutput) {
+      if (error.stdout.length === 0 && error.stderr.length === 0) {
+        console.error('No stdout or stderr output from process')
+      } else {
+        if (error.stdout.length > 0) {
+          console.error(error.stdout)
+        }
+
+        if (error.stderr.length > 0) {
+          console.error(error.stderr)
+        }
+      }
```

Comment:
We may wish to format this a bit for processes that spit out info in both stdout/stderr but for now I'm just dumping everything.